### PR TITLE
added correct files for 2.4r2 pin mod and Y endstop relocation mod

### PR DESCRIPTION
**Pull request by hartk1213**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Further notes
@MapleLeafMakers here is the PR you were looking for , let me know if its missing anything or needs any changes, im pretty sure i have everything there that needs to be there

### CODE REVIEW

**Review by newphreak — APPROVED** [review]
LGTM

**Comment by MapleLeafMakers** [comment]
Looks like a typo in an image filename here: 
image

**Comment by hartk1213** [comment]
@newphreak @MapleLeafMakers should be good to go now

File: printer_mods/hartk1213/Voron2.4_Trident_Pins_Mod/README.md [hunk 1]
```diff
@@ -34,6 +35,31 @@ here is what the XY Joint will look like
 ![XY Joint](Images/1.jpg)
 ![XY Joint](Images/2.jpg)
 
+There is a new file for the A Drive to go along with [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12) mod that has the X endstop on the toolhead
```
Comment (cbc02009) [inline printer_mods/hartk1213/Voron2.4_Trident_Pins_Mod/README.md:38]: Can you make this a relative link? just makes it less likely to break if something changes.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Review by cbc02009 — APPROVED** [review]

File: printer_mods/hartk1213/Voron2.4_Trident_Pins_Mod/README.md (hunk 1 above)
Comment (hartk1213) [reply printer_mods/hartk1213/Voron2.4_Trident_Pins_Mod/README.md]: just removed it and replaced it with 2.4r2/trident carriage

**Comment by Eifel-Joe** [comment]
Hello, I know I'm very late with my objection, but I just stumbled across that in the R2 update of this mod, flange bearings are used on xy-joints instead of toothed idlers. When I took a look at the CAD, more precisely the section analysis, I noticed that the flanks of the flange bearings are within the belt.
This means that the belt is pressed closer to the extrusion by the flange bearings, which means that the belt no longer runs parallel to the extrusion as it would have done with an idler. It will be diagonally towards the carriage block.
Isn't this a mistake? In my opinion, the belts should run exactly parallel to the extrusion.
Shouldn't the position of the pin for the use of flange bearings be moved at this point?
bearing
idler

**Pull request merged**